### PR TITLE
pkgdown 1.4

### PR DESCRIPTION
**Pull request by hadley**
@jayhesselberth does this look ok to you? Is there anything we could do to give you more credit for co-maintaining the package?

### CODE REVIEW

File: content/articles/2019-09-pkgdown-1-4-0.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - pkgdown
+---
+
+We're happy to announce that [pkgdown](https://pkgdown.r-lib.org/) 1.4.0 is now
+available on CRAN. pkgdown is designed to make it quick and easy to build a
+website for your package. 
+
+## Development and deployment modes
+
+The main change in this version is that `build_site()`, `build_reference()` and  `build_home()` gain a `devel` parameter that controls whether you're in deployment or development mode.
```
Comment (batpigandme) [inline content/articles/2019-09-pkgdown-1-4-0.Rmarkdown]: Is there magical linking here? If not, I usually link to the function reference on first mention.

File: content/articles/2019-09-pkgdown-1-4-0.Rmarkdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    If you're using `build_site()` as part of an existing deployment workflow, 
+    you may need to set `install = FALSE` so that the package doesn't get 
+    installed twice.
+
+*   `devel = TRUE` activates development mode, simulates package loading with
+    `pkgload::load_all()` and does all work in the same process. This makes it
+    most suitable for rapid iteration, and so is the default for 
+    `build_reference()`. 
+  
+There were two related changes to `build_reference()` and `build_home()`: `build_reference()` no longer runs `devtools::document()` and `build_home()` no longer re-builds `README.Rmd`.  This makes pkgdown's responsibilities more clear: it only modifies files in `doc/`.
```
Comment (batpigandme) [inline content/articles/2019-09-pkgdown-1-4-0.Rmarkdown]:
```suggestion
There were two related changes to `build_reference()` and `build_home()`: `build_reference()` no longer runs `devtools::document()`, and `build_home()` no longer re-builds `README.Rmd`.  This makes pkgdown's responsibilities clearer: it only modifies files in `doc/`.
```

Comment (batpigandme) [reply content/articles/2019-09-pkgdown-1-4-0.Rmarkdown]: Is this a conscious use of hyphenated re-build to harken to the function names? If not, I think rebuild can be used in closed form.

File: content/articles/2019-09-pkgdown-1-4-0.Rmarkdown [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Other important changes
+
+* `build_home()` now strips quotes from `Title` and `Description` fields 
+  when generating page metadata. Additionally, you can now override the 
+  defaults via the `title` and `description` fields in the `home` section of 
+  `_pkgdown.yml`. Setting a custom title is good practice as it allows you to 
+  add "R" and "package" to the title, which is likely to be useful for SE.
```
Comment (batpigandme) [inline content/articles/2019-09-pkgdown-1-4-0.Rmarkdown]:
```suggestion
  add "R" and "package" to the title, which is likely to be useful for SEO.
```

File: content/articles/2019-09-pkgdown-1-4-0.Rmarkdown [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+* [`vignette("linking")`](https://pkgdown.r-lib.org/articles/linking.html) 
+  describes how pkgdown's automatic linking works, and how to ensure that
+  cross-package links point to the right place.
+
+* The rendering of `\examples{}`, `\dots`, `\tabular{}`, `\subsection{}`,
+  and `\preformatted{}` has been improved.
+
+* The navbar is now automatically hidden with 
+  [headroom.js](https://wicky.nillia.ms/headroom.js/) and the sidebar
```
Comment (batpigandme) [inline content/articles/2019-09-pkgdown-1-4-0.Rmarkdown]:
```suggestion
  [headroom.js](https://wicky.nillia.ms/headroom.js/), and the sidebar
```

File: content/articles/2019-09-pkgdown-1-4-0.Rmarkdown (hunk 1 above)
Comment (batpigandme) [reply content/articles/2019-09-pkgdown-1-4-0.Rmarkdown]: I'd be willing to waive that preference in this case, since it's pretty function-laden prose.

**Comment by jayhesselberth** [comment]
Looks good to me.

**Comment by batpigandme** [comment]
@hadley — do you want to look at my eds here? There are two minor questions involved…
(I don't totally grok the difference between Review requests and Assign 😳).

**Comment by hadley** [comment]
Yeah, I'll try and get to later today.

File: content/articles/2019-09-pkgdown-1-4-0.Rmarkdown (hunk 2 above)
Comment (jayhesselberth) [inline content/articles/2019-09-pkgdown-1-4-0.Rmarkdown]: Should be `docs/` not `doc/`.

File: content/articles/2019-09-pkgdown-1-4-0.markdown [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    installed twice.
+
+*   `devel = TRUE` activates development mode, simulates package loading with
+    `pkgload::load_all()` and does all work in the same process. This makes it
+    most suitable for rapid iteration, and so is the default for 
+    `build_reference()`. 
+  
+There were two related changes to `build_reference()` and `build_home()`: `build_reference()` no longer runs `devtools::document()` and `build_home()` no longer re-builds `README.Rmd`.  This makes pkgdown's responsibilities more clear: it only modifies files in `doc/`.
+
+## Other important changes
```
Comment (jayhesselberth) [inline content/articles/2019-09-pkgdown-1-4-0.markdown]: I think the move to rdrr.io for documentation should be in the "important changes" section.

**Comment by batpigandme** [comment]
> Yeah, I'll try and get to later today.

No rush. Just didn't quite know how to mentally parse "assign"

**Comment by hadley** [comment]
Thanks all!

**Pull request merged**